src/cirrus/lib/statedb.py

```diff
@@ -134,12 +135,31 @@ def get_items_page(self, collections_workflow: str,
         items = {
             'items': []
         }
+        query_kwargs = {
+            'collections_workflow': collections_workflow,
+            'state': state,
+            'since': since,
+            'sort_ascending': sort_ascending,
+            'sort_index': sort_index,
+            'Limit': limit,
+        }
+
         if nextkey:
             dbitem = self.get_dbitem(nextkey)
             startkey = { key: dbitem[key] for key in ['collections_workflow', 'itemids', 'state_updated', 'updated']}
-            resp = self.query(collections_workflow, state=state, since=since, sort_ascending=sort_ascending, sort_index=sort_index, Limit=limit, ExclusiveStartKey=startkey, )
-        else:
-            resp = self.query(collections_workflow, state=state, since=since,  sort_ascending=sort_ascending, sort_index=sort_index, Limit=limit)
+            query_kwargs.update({
+                'ExclusiveStartKey': startkey
+                })
```

Comment:
This is an unusual way to modify a dictionary to me. I think you would want something more like:

```suggestion
            query_kwargs['ExclusiveStartKey'] = startkey
```

Reply:
Yeah, I think I'd done it that way so I could build up a sub-dictionary to feed into `update`, but you're right -- not needed, better to use the standard approach.